theme.js

```diff
@@ -2863,4 +2890,126 @@ export default (variables = defaultThemeVariables) => ({
       },
     },
   },
+
+  'shoutem.ui.InlineDropDownMenu': {
+    container: {
+      paddingTop: 12,
+      paddingHorizontal: variables.mediumGutter,
+      paddingBottom: 4,
+      backgroundColor: variables.paperColor,
+    },
+    icon: {
+      color: variables.text.color,
+    }
+  },
+
+  'shoutem.ui.InlineDropDownMenuItem': {
+    container: {
+      paddingTop: 10,
+      paddingHorizontal: variables.mediumGutter,
+      paddingBottom: 4,
+      backgroundColor: variables.paperColor,
+      borderTopWidth: 1,
+      borderTopColor: variables.backgroundColor,
+    },
+  },
+
+  'shoutem.ui.TabMenu': {
+    container: {
+      paddingHorizontal: variables.smallGutter,
+      backgroundColor: variables.backgroundColor,
+      height: 44,
+    },
+  },
+
+  'shoutem.ui.TabMenuItem': {
+    tabulator: {
+      backgroundColor: variables.text.color,
+      height: 1,
+      borderRadius: 1,
+      flexDirection: 'row',
+      marginBottom: 8,
+      marginLeft: 8,
+    },
+    text: {
+      marginTop: 12,
+      marginHorizontal: 8,
+      marginBottom: 12,
+      opacity: 0.3,
+    },
+    selectedText: {
+      marginBottom: 4,
+      opacity: 1,
+    }
+  },
+
+  'shoutem.ui.YearPickerButton': {
+    container: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      backgroundColor: '#F9F9F9',
+      borderWidth: 1,
+      borderRadius: 8,
+      borderColor: 'rgba(130, 130, 130, 0.1)',
+      paddingLeft: 15,
+      paddingVertical: 8,
+      paddingRight: 8,
+      marginRight: 8,
+    },
+    icon: {
+      color: variables.text.color,
+    }
+  },
+
+  'shoutem.ui.YearPickerModal': {
+    container: {
+      padding: 8,
+      backgroundColor: '#F9F9F9',
```

Comment:
Should we maybe use colors that can be modified via theme variables i.e. paper color, background color, etc.? Rather than hardcoding and then any changes wanted require your custom theme to directly modify `shoutem.ui.YearPickerModal` instead of just syncing up with your theme's `paperColor`?

Let me know if my question doesn't make sense, I'm not sure I worded it the best.